### ISSUE TITLE
Question regarding fine tuning

### ISSUE THREAD

**Issue by Rabbott93** [experiencor/basic-yolo-keras#212]
@experiencor 
In the Red blood cells example, if I want to use the weights produced (weights_blood.h5) to fine tune to another similar task, how would I go about doing this?
I understand that the yolo.weights are the backend and  weights_blood.h5 are the backend and frontend. Therefore, I would need a way of extracting the frontend from the weights_blood.h5 file in order to fine tune to a new task?

Many Thanks Rach

**Comment by experiencor** [comment]
@Rabbott93 You can load weights_blood.h5 as the initial weights and start from here. During training, both backend and frontend get trained, not just the frontend.

PS 1: yolo.weights includes both frontend and backend. 

PS 2: provide backend weights because this part is the same regardless of the number of classes.

**Comment by Rabbott93** [comment]
@experiencor 

Thank you for your quick reply! 

When I use  the yolo.weights file, training works. However, when I use my weights_object.h5 file for fine tuning, the loss  gives a nan value and recall=0. 

![screenshot from 2018-03-28 10-10-37](https://user-images.githubusercontent.com/26873659/38019843-97a1ef68-3270-11e8-84b2-b693fd3bc970.png)

Do you know why this is?

Thanks!
